### PR TITLE
Populating Foundation pages metadata

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

To expand searchable surface, populates the sections and tags metadata for Foundation pages.

Related to https://github.com/grommet/hpe-design-system/issues/1849

#### Where should the reviewer start?

data/structures/foundation.js

#### What are the relevant issues?

Related to https://github.com/grommet/hpe-design-system/issues/1849

### CODE REVIEW

File: aries-site/src/pages/foundation/human-centered.mdx [hunk 1]
```diff
@@ -10,4 +10,4 @@ Being human-centered requires attention to the needs of others. We first listen
...
+The HPE Design System elicits a conversation in two ways. It first enables people to craft experiences that engage in an interactive dialogue. The Design System allows for the design to express tone and give voice to an interaction. Each interaction elicits a positive or negative response that compiles into an entire experience for the user. Second, the Design System elicits conversation to continue its evolution, growth, maturity, and adaptability. It will not meet every need and solve every solution. However, it can bring about a conversation to safely allow people to be bold and accelerate what's next.
```
Comment (britt6612) [inline aries-site/src/pages/foundation/human-centered.mdx:13]: nice catch!

**Comment by britt6612** [comment]
Are you planning on breaking out adding metadata on all the pages ?

File: aries-site/src/data/structures/foundation.js [hunk 2]
```diff
@@ -17,7 +17,23 @@ export const foundation = [
     },
     seoDescription: `Accessibility is central to the HPE Design System and 
     ensures that HPE applications are usable by as many people as possible.`,
-    sections: [],
+    sections: ['Accessibility in design', 'Accessibility in development'],
```
Comment (britt6612) [inline aries-site/src/data/structures/foundation.js:20]: would Resources not be part of the sections?

Comment (halocline) [reply aries-site/src/data/structures/foundation.js:20]: Right now, things like 'guidance', 'resources', 'variants' will be too generic for our search and end up polluting the results. Holding off for now.

**Comment by halocline** [comment]
> Are you planning on breaking out adding metadata on all the pages ?

I have a "Components" PR in the works. Should be up by lunch time.

File: aries-site/src/data/structures/foundation.js (hunk 2 above)
Comment (britt6612) [reply aries-site/src/data/structures/foundation.js:20]: That's a good point!

**Comment by britt6612** [comment]
> > Are you planning on breaking out adding metadata on all the pages ?
> 
> I have a "Components" PR in the works. Should be up by lunch time.

Sounds good! This LGTM! 
Thank you for putting this together!

**Pull request merged**